src/components/orders/GasFeeDisplay/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { formatSmart, safeTokenName } from '@gnosis.pm/dex-js'
+
+import { Order } from 'api/operator'
+
+import { ONE_BIG_NUMBER, TEN_BIG_NUMBER } from 'const'
+
+const Wrapper = styled.div`
+  & > span {
+    margin: 0 0.5rem 0 0;
+  }
+`
+
+const UsdAmount = styled.span`
+  color: ${({ theme }): string => theme.grey};
+`
+
+export type Props = { order: Order }
+
+export function GasFeeDisplay(props: Props): JSX.Element | null {
+  const {
+    order: { feeAmount, executedFeeAmount, sellToken, sellTokenAddress },
+  } = props
+
+  // TODO: fetch amount in USD
+  const executedFeeUSD = '0.99'
+  const totalFeeUSD = '0.35'
```

Comment:
this one is the `executedFeeUsd`, right?

Reply:
Right, I inverted them. 
Still, fake data for now, will take care of it when I add the usd amount calculation